src/tasks/openCoverage.js

```diff
@@ -0,0 +1,27 @@
+const { platform } = require('os');
+const {
+  promises: { stat },
```

Comment:
Perhaps use something else so we don't get the `ExperimentalWarning: The fs.promises API is experimental` error.

Reply:
I also don't get that error